src/auspex/filters/io.py

```diff
@@ -152,6 +173,13 @@ def save_json(self):
 
         self.data_group = self.group.create_group("data")
 
+        # If desired, push experimental metadata into the h5 file
```

Comment:
Do we even want the option of saving JSON at this point?

Reply:
I don't think so